content/en/tracing/profiling/_index.md

```diff
@@ -129,6 +129,59 @@ The Datadog Profiler requires Python 2.7+. Memory profiling only works on Python
 [3]: /tracing/visualization/#services
 {{% /tab %}}
 
+{{% tab "Go" %}}
+
+The Datadog Profiler requires Go 1.12+. To begin profiling applications:
+
+1. Install `ddtrace` which contains both tracing and profiling:
+
+    ```shell
+    go get gopkg.in/DataDog/dd-trace-go.v1/ddtrace
+    ```
+
+     **Note**: Profiling is available in the `ddtrace` library for versions 1.23.0+.
+
+2. Set `DD_ENV`, `DD_SERVICE`, and `DD_VERSION` in your environment variables.
+
+3. Import the [Go profiler][5] at the start of your application
+
+    ```Go
+    import "gopkg.in/DataDog/dd-trace-go.v1/profiler"
+    ```
+
+4. To profile your code, add a [Datadog API key][1], then start the profiler:
+
+    ```Go
+    err := profiler.Start(
```

Comment:
It's odd to recommend a mix of env vars and code config. Can we choose one or the other as the primary flow and make the other secondary?

Reply:
Customers should be used to ENV variables on tracing.
The API key is not supprted with ENV variables today

Reply:
https://docs.datadoghq.com/tracing/setup/go/#configuration shows quickstart go tracing configured via code, not env var.